rules/prefer-spread.js

```diff
@@ -14,7 +14,32 @@ const selector = [
 ].join('');
 
 const create = context => {
-	const getSource = node => context.getSourceCode().getText(node);
+	const sourceCode = context.getSourceCode();
+	const getSource = node => sourceCode.getText(node);
+
+	const needsSemicolon = node => {
```

Comment:
Should this be in `util`?

Reply:
I tried to make a better util function for this, but seems need handle differently in different case.I don't think current implementation can use in other rule, at least not yet.

Reply:
Ok. Just something to keep in mind in case we need it in multiple places.